### ISSUE TITLE
Everyone should be able to log out

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#1714]
At the moment the logout button can be hidden from users based on permissions. This poses a security risk. It was removed because users were accidentally clicking it on mobile and then were unable to log back in. Find a way to show the logout button but make it hard to click by accident.

**Comment by SCdF** [comment]
@mandric please close or schedule before the end of this sprint. See [triaging old issues](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues).

**Comment by nice-snek** [comment]
hi frand @SCdF, please [triage](https://github.com/medic/medic-docs/blob/master/development/workflow.md#triaging-old-issues) before the end of this sprint.

**Comment by SCdF** [comment]
As a security issue even though no one cares enough to fix it we should still keep it around.

**Comment by sglangevin** [comment]
Yeah, we will get to this one. At the very least, we will include it with our roles & permissions update.
